flink-runtime/src/main/java/org/apache/flink/runtime/rpc/jobmaster/JobMaster.java

```diff
@@ -230,153 +203,14 @@ public Acknowledge updateTaskExecutionState(TaskExecutionState taskExecutionStat
 	 */
 	@RpcMethod
 	public void registerAtResourceManager(final String address) {
-		currentRegistrationRun = UUID.randomUUID();
-
-		Future<ResourceManagerGateway> resourceManagerFuture = getRpcService().connect(address, ResourceManagerGateway.class);
-
-		handleResourceManagerRegistration(
-			new JobMasterRegistration(getAddress()),
-			1,
-			resourceManagerFuture,
-			currentRegistrationRun,
-			initialRegistrationTimeout,
-			maxRegistrationTimeout,
-			registrationDuration.fromNow());
+		//TODO:: register at the RM
```

Comment:
We usually don't use TODOs in our commits. Experience shows that they are usually forgotten.